typeDefinition/CookieChain.d.ts

```diff
@@ -71,5 +71,5 @@ interface CookieEvalModifiers<T> extends
 
 interface CookieAbandonedChain extends AbstractChain {}
 
-type CookieQueryResult = string | void;
+type CookieQueryResult = string | undefined;
 type CookieEvalResult = boolean | void;
```

Comment:
@JeKachan seems we have other voids

Reply:
Making the same has no sense for boolean values. But make sense do it for keeping consistency.